### DISCUSSION TITLE
blog에 이미지 업로드

### DISCUSSION THREAD

**Discussion by whatareyoudoingz** [chemistryx/hyde#14] [Q&A]
안녕하세요! 
저는 ChemistryX님이 제공해주시는 테마를 적용해서 사용하고 있는 사용자입니다.
하다보니 blog에 올린 게시물 안에 사진(이미지) 삽입이 안 되던데 어떻게 해야 사진이 업로드가 되나요??ㅠㅠ

**Comment by chemistryx** [comment]
안녕하세요.
게시물 내 이미지의 경우 HTML 태그를 활용하거나, Markdown 문법을 통해 추가할 수 있습니다.

### HTML 태그를 통한 삽입
```html
<img src="경로" />
```

### Markdown 문법을 통한 삽입
```md
![이미지 설명](경로)
```

위 방법들로도 이미지가 보이지 않으신다면 경로 문제일 가능성이 높으니 절대 경로나, 외부 이미지 Provider를 통해 삽입해보시길 바랍니다.

감사합니다.

**Reply by whatareyoudoingz** [reply]
해결했습니다! 감사합니다☺️
